Python/specialize.c

```diff
@@ -0,0 +1,136 @@
+
+#define PY_LOCAL_AGGRESSIVE
```

Comment:
Why turn this on here? That makes debugging this file (with e.g. gdb or VS) harder.

Reply:
I'll remove it, LTO should inline stuff for us anyway.